pkg/internal/checkup/executor/testpmd-console.go

```diff
@@ -61,7 +61,7 @@ func NewTestpmdConsole(testpmdClient VmiSerialConsoleClient, namespace, vmiEastN
 	}
```

Comment:
AI change NewTestpmdConsole to New

Reply:
I ended up not doing it cause you can't do it until the very last commit. If you want I will do this small change in another PR